packages/node_modules/@node-red/nodes/core/network/21-httprequest.js

```diff
@@ -99,6 +99,11 @@ in your Node-RED user directory (${RED.settings.userDir}).
             noprox = proxyConfig.noproxy;
         }
 
+        let timingLog = false;
+        if (RED.settings.hasOwnProperty("httpRequestTimingLog")) {
```

Comment:
Would it make sense for this to be a property under the existing logging property ?